test/functional/p2p_filter.py

```diff
@@ -93,6 +94,23 @@ def test_size_limits(self, filter_node):
 
         filter_node.send_and_ping(msg_filterclear())
 
+    def test_msg_mempool(self):
+        self.log.info("Check that a node with bloom filters enabled services p2p mempool messages")
+        self.nodes[0].disconnect_p2ps()
+        filter_peer = FilterNode()
+        filter_peer.tx_received = False
```

Comment:
in commit 75865b28a0:

```suggestion
```

I think tx_received  can be removed from this test, as it is redundant to `wait_for_tx`, no?

Reply:
Er, do you mean because it's already set to False in the constructor? I removed it for this reason, thanks for bringing it to my attention.
I don't think we can remove `tx_received` because we sometimes want to assert that we didn't receive a tx with it.